### PR TITLE
Issue 14118 relate content improvements

### PR DESCRIPTION
**Pull request by jdotcms**

### CODE REVIEW

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.jsp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div id="${id}">
-	<div dojoAttachPoint="dialog" dojoType="dijit.Dialog" dojoAttachEvent='onHide:_clearSearch' class="related-content-dialog">
+	<div dojoAttachPoint="dialog" dojoType="dijit.Dialog" dojoAttachEvent='onHide:_clearSearch' class="related-content-dialog" style="width:90%;height:90%">
```
Comment (jgambarios) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.jsp]: The style changes should be inside the `related-content-dialog` class

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp [hunk 2]
```diff
@@ -1,20 +1,22 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>
-<form dojoAttachPoint="search_form" onsubmit="return false;" id="searchForm" class="related-content-form">
-    <div dojoType="dijit.layout.BorderContainer" design="sidebar" gutters="false" liveSplitters="true" dojoAttachPoint="borderContainer" class="related-content-container">
+<form dojoAttachPoint="search_form" onsubmit="return false;" id="searchForm" >
+<div style="display:flex;flex-direction: row;">
+	
 
         <!-- START Left Column -->
-        <div dojoType="dijit.layout.ContentPane" splitter="false" region="leading" class="portlet-sidebar-wrapper">
+        <div class="portlet-sidebar-wrapper" style="width:200px;min-height:700px;">
```
Comment (jgambarios) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: The style changes should be inside the `portlet-sidebar-wrapper` class

**Review by jgambarios — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.jsp (hunk 1 above)
Comment (jdotcms) [reply dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.jsp]: done

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp (hunk 2 above)
Comment (jdotcms) [reply dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp]: done

**Review by jgambarios — APPROVED** [review]

**Comment by wezell** [comment]
Possibly causing the "Reuse content window" to get squished:

screen shot 2018-07-23 at 1 43 22 pm

**Comment by john-thomas-dotcms** [comment]
In general, all our popup-style windows in 5.0 are way too squished.

I could rant quite a bit about this, because it's far more annoying than you can ever know if you don't use the application daily :).  But bottom line, the wasted space on the top and bottom is already a problem, and the wasted space on the right and left is obviously an issue here, and will become even more of a problem if we move to allow placing content type fields in a grid (as has been discussed).

image

So my suggestion is to just widen the "pop-up" so it leaves only a little unused space on the right and left sides.  This widening alone may solve this specific issue, and will also go a long way towards resolving the general issue with the unnecessarily small popup-type screens.

image

**Pull request merged**